### PR TITLE
KDB.AI 1.1 release update

### PR DESCRIPTION
**Pull request by sancoap**
Added support for hybrid search, sparse vectors, BM25, LlamaIndex & HuggingFace Integration

### CODE REVIEW

File: docs/tools/vdb_table/data/kdbai.json [hunk 1]
```diff
@@ -97,7 +97,7 @@
     "comment": ""
   },
   "llamaindex": {
-    "support": "",
+    "support": "full",
     "source_url": "",
```
Comment (dhruv-anand-aintech) [inline docs/tools/vdb_table/data/kdbai.json:101]: please add source_url

File: docs/tools/vdb_table/data/kdbai.json [hunk 2]
```diff
@@ -107,7 +107,7 @@
     "comment": ""
   },
   "pricing": {
-    "value": "Cloud = Free, limited memory/storage\n\nOn-Prem = Contact Sales",
+    "value": "Cloud = Free, OnPrem = 90-day trial, contact sales",
```
Comment (dhruv-anand-aintech) [inline docs/tools/vdb_table/data/kdbai.json:110]: might be better to add "Free: 4 GB memory per instance, 30 GB data storage"

File: docs/tools/vdb_table/data/kdbai.json [hunk 3]
```diff
@@ -67,17 +67,17 @@
...
   },
   "embeddings_image": {
-    "support": "",
+    "support": "full",
     "source_url": "",
     "comment": ""
   },
   "embeddings_structured": {
-    "support": "",
+    "support": "full",
```
Comment (dhruv-anand-aintech) [inline docs/tools/vdb_table/data/kdbai.json:80]: this attribute is to do with support for graph embeddings, is that something KDB.AI has?

File: docs/tools/vdb_table/data/kdbai.json [hunk 4]
```diff
@@ -67,17 +67,17 @@
...
   },
   "embeddings_text": {
-    "support": "",
+    "support": "full",
     "source_url": "",
     "comment": ""
   },
   "embeddings_image": {
-    "support": "",
+    "support": "full",
```
Comment (dhruv-anand-aintech) [inline docs/tools/vdb_table/data/kdbai.json:75]: please add url of the relevant documentation

**Review by dhruv-anand-aintech — COMMENTED** [review]
Please add urls of the relevant documentation for all the new support claims. Thanks

**Comment by sancoap** [comment]
> Please add urls of the relevant documentation for all the new support claims. Thanks

Updated source_url with documentation links

**Review by sancoap — COMMENTED** [review]
hi Dhruv, please let me know if any additional information is needed. Kindly approve after review.

**Review by dhruv-anand-aintech — APPROVED** [review]

**Pull request merged**